app/scripts/modules/core/src/pipeline/config/triggers/baseBuild/BaseBuildTrigger.tsx

```diff
@@ -34,11 +34,13 @@ export function BaseBuildTrigger(buildTriggerProps: IBaseBuildTriggerConfigProps
     }
   }, [fetchJobs.result]);
 
+  const label = buildTriggerType === 'jenkins' ? 'Controller' : 'Master';
```

Comment:
+1 can we check what terminology the other build systems use?

Reply:
Sure, I will look into this and circle back with you both. Right now, Jenkins, Travis and Wercker use this component and were all referring to "master".

Reply:
@ezimanyi @christopherthielen I don't see any strong ties to `Controller` terminology with these other trigger types. I find `Build Service` more intuitive for Travis, Concourse, and Wercker since they are not tied to the Controller terminology. What do you think?

Reply:
I like "Build Service"; that's what I used on the back-end when I was doing some [refactoring of Igor](https://github.com/spinnaker/igor/pull/378/commits/972f478a0db4b11d96f2ee53eaa7dcc0eb55dcd0) and wanted to improve the terminology so this will match the back-end.

Reply:
Ok I will update the remaining triggers, but leave Jenkins as `Controller`